src/material/slide-toggle/testing/slide-toggle-harness-filters.ts

```diff
@@ -16,4 +16,6 @@ export interface SlideToggleHarnessFilters extends BaseHarnessFilters {
   name?: string;
   /** Only find instances with the given checked value. */
   checked?: boolean;
+  /** Only find instances that are disabled */
```

Comment:
I don't think that this is accurate. This will look up instances based on their disabled status. E.g. you could look only for enabled slide toggles.